index.html

```diff
@@ -0,0 +1,1458 @@
+            d="M368.907 465.499C370.426 463.69 371.498 461.547 372.036 459.246C372.574 456.945 372.562 454.549 372.001 452.254C371.441 449.958 370.348 447.826 368.81 446.032C367.273 444.237 365.335 442.829 363.152 441.923C360.97 441.017 358.604 440.637 356.248 440.815C353.891 440.993 351.61 441.724 349.588 442.947C347.567 444.171 345.861 445.853 344.611 447.859C343.36 449.864 342.6 452.135 342.39 454.489L348.841 455.063C348.96 453.722 349.393 452.428 350.106 451.285C350.818 450.143 351.79 449.185 352.941 448.487C354.093 447.79 355.393 447.374 356.735 447.273C358.078 447.172 359.425 447.388 360.669 447.904C361.912 448.42 363.016 449.222 363.892 450.245C364.768 451.267 365.391 452.481 365.71 453.789C366.029 455.097 366.036 456.462 365.73 457.773C365.424 459.084 364.813 460.304 363.947 461.335L368.907 465.499Z"
+            fill="#EBEBFF"
+          />
+          <path
+            d="M352.497 429C352.278 432.194 353.337 435.344 355.441 437.757C357.544 440.17 360.521 441.649 363.715 441.868C366.909 442.086 370.059 441.027 372.472 438.924C374.885 436.82 376.364 433.844 376.582 430.65L352.497 429Z"
+            fill="#6070FF"
+          />
+          <rect
+            y="37.4871"
+            width="51.9898"
+            height="9.30073"
+            transform="rotate(-46.1415 0 37.4871)"
+            fill="#EBEBFF"
+          />
+          <path
+            d="M22.1907 24.642C20.1859 29.3378 20.1285 34.6377 22.0312 39.3758C23.934 44.1139 27.641 47.902 32.3368 49.9069C37.0326 51.9118 42.3324 51.9692 47.0705 50.0664C51.8086 48.1636 55.5968 44.4566 57.6016 39.7609L22.1907 24.642Z"
+            fill="#6070FF"
+          />
+          <circle cx="255.5" cy="13.5" r="7" fill="white" stroke="#CBCDFF" />
+          <circle cx="337.5" cy="216.5" r="5" fill="white" stroke="#CBCDFF" />
+          <circle cx="94.5" cy="548.5" r="7" fill="white" stroke="#CBCDFF" />
+          <path
+            d="M455.241 74.9701C451.195 67.7876 445.514 61.659 438.659 57.0803C431.803 52.5017 423.966 49.6018 415.782 48.6154C407.597 47.6291 399.295 48.584 391.548 51.4028C383.802 54.2216 376.827 58.8252 371.191 64.8407C365.554 70.8562 361.413 78.1145 359.103 86.0281C356.793 93.9416 356.379 102.288 357.895 110.391C359.411 118.494 362.813 126.127 367.828 132.67C372.842 139.214 379.326 144.485 386.756 148.056L396.543 127.695C392.31 125.661 388.616 122.658 385.759 118.93C382.903 115.202 380.964 110.854 380.101 106.238C379.237 101.621 379.473 96.8662 380.789 92.3579C382.105 87.8496 384.464 83.7145 387.675 80.2874C390.886 76.8604 394.86 74.2378 399.273 72.6319C403.686 71.026 408.416 70.482 413.079 71.044C417.741 71.6059 422.206 73.258 426.112 75.8664C430.017 78.4748 433.254 81.9663 435.559 86.0582L455.241 74.9701Z"
+            fill="#6070FF"
+          />
+          <rect
+            x="328"
+            y="61.79"
+            width="95.5804"
+            height="22.021"
+            transform="rotate(-30 328 61.79)"
+            fill="#EBEBFF"
+          />
+        </svg>
```

Comment:
[Optional] I recommend using CSS background image properties to set the background image instead of using SVG for simplicity, maintenance, and performance. 👍